firsttris/vscode-jest-runner
# add exactTestName option

### Background
Hi 👋 Power-user and big fan of the extension!

I often run into the issue of overlapping test names.
e.g. One test called "do something" and another called "do something else"; Running the first test using vscode-jest-runner will run both tests. This is because Jest's `-t` option regex matches. Most of the time I'd like to run the single exact test name. In the off chance I'm looking to run multiple tests, I could still go into the terminal and edit the command, but in 95% of cases I'm looking for one test and can be confused when an unexpected output is shown (until I realize multiple tests are running).

### Changes
- add the `exactTestName` test name boolean config option
  - which wraps the test name in `^` `$` for an exact regex match
  - add option to `package.json`
  - add option to `README.md`
- remove some unnecessary whitespace _(my editor is set-up to do this automatically, lmk if I should re-add it)_

**FYI:** _I have verified that the wrapping works in my terminal, by manually replaying a generated terminal command from vscode-jest-runner and adding the characters — but, I haven't built the extension and tested end-to-end._
